service/dap/server_test.go

```diff
@@ -2553,6 +2556,7 @@ func TestBadLaunchRequests(t *testing.T) {
 		client.LaunchRequest("debug", fixture.Path+"_does_not_exist", stopOnEntry)
 		expectFailedToLaunch(client.ExpectErrorResponse(t)) // Build error
 
+		// Test for the DAP-specific detailed error message.
```

Comment:
Please move this back. The earlier comment was about moving the new test.

Reply:
Now I got the clarification, switched back with the test